share/picongpu/examples/Bunch/include/picongpu/param/fieldBackground.param

```diff
@@ -58,8 +66,10 @@ namespace picongpu
                   0.8e-6,
                   /* pulselength [s], sigma of std. gauss for intensity (E^2) */
                   10.0e-15 / 2.3548200450309493820231386529194,
+#ifndef PARAM_TWTSFAST
```

Comment:
Please do not use `#ifdef` and `#ifndef` to control the code path. 
Please use `#if PARAM_TWTSFAST == 0` to check that it is disabled and `#if PARAM_TWTSFAST == 1` for enabled